roles/common/defaults/main.yml

```diff
@@ -104,6 +104,9 @@ postgres_configuration_secret: '{{ ansible_operator_meta.name }}-postgres-config
 # Secret to lookup that provide the PostgreSQL configuration for old database
 postgres_migrant_configuration_secret: ''
 
+# Enable PostgreSQL SCRAM-SHA-256 migration
+postgres_scram_migration_enabled: true
```

Comment:
Should we add `postgres_scram_migration_enabled` to the CRD ? otherwise users won't be able to turn this off

Reply:
I can't think of a scenario when a user would want md5 over sha-scram-256.  I think we can just default to true and not make it customizable.